pkgs/applications/networking/nextcloud-client/default.nix

```diff
@@ -15,20 +15,22 @@
 , qtquickcontrols2
 , qtgraphicaleffects
 , sqlite
+, qtwebsockets
 }:
 
 mkDerivation rec {
   pname = "nextcloud-client";
-  version = "3.0.3";
+  version = "3.1.3";
 
   src = fetchFromGitHub {
     owner = "nextcloud";
     repo = "desktop";
     rev = "v${version}";
-    sha256 = "0idh8i71jivdjjs2y62l22yl3qxwgcr0hf53dad587bzgkkkr223";
+    sha256 = "sha256-8Ql6tOvWOjAvMJA87WlT9TbpnbciBsjDxRuYlMVi/m8=";
   };
 
   patches = [
+    # Explicitly copy dbus files into the store dir
```

Comment:
That comment doesn't say anything that the file name doesn't already say.

Reply:
Yep, but it helps with bots and fmts.

Reply:
Just adding it for the bot is not the intention. A next version of hammering should ignore this is the patch had a file name.

Reply:
About the bot, I was just joking, I know that having things minimally documented is better than just leaving it there.

Reply:
Maybe it could be mentioned _why_ this patch is being applied? Is it because dbus is not available at build time? (Just guessing)

Reply:
I took a quick look and thought the name lives up to lol

Reply:
I agree with @turion that the comment just repeats the self-describing file name and doesn't add any value.

Please replace it with an explanation of why the patch is necessary, e.g. based on the corresponding commit's message (8d62a398a8460e1e89d8d9baf2c5b9c2aa75b9b6):

    Also move explicitly dbus configuration files to the store path rather than
    `/etc/dbus-1/services`.

Reply:
(To be fair, the patch was already present before, and not changed in this PR.)